test/testing/fakeauth_test.js

```diff
@@ -27,7 +28,7 @@ testApi.declare({
 
 
 suite('fakeauth', function() {
-  var fakeauth = require('../../src/fakeauth');
+  var fakeauth = require('../../lib/fakeauth');
```

Comment:
lib.  The reason is that if we use src, we need to use the babel register hook to transpile everything on runtime.  This is bad because we are then no longer testing the code that we ship.

The example that I think jonas is referring to is that we found that Array.prototype.includes just can't be done except through runtime transpilation.